packages/manager/modules/bm-server-components/src/network-tile/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Haven't the translation file for the other languages been moved here?

Reply:
Its not mandatory. I will push to translation and it will be auto populated.